plugins/techdocs-backend/src/techdocs/stages/generate/techdocs.ts

```diff
@@ -46,35 +69,46 @@ export class TechdocsGenerator implements GeneratorBase {
     const resultDir = fs.mkdtempSync(
       path.join(tmpdirResolvedPath, 'techdocs-tmp-'),
     );
-
+    const [log, logStream] = createStream();
+    console.log(this.options.runGeneratorIn);
```

Comment:
We can probably remove this `console.log`.

Reply:
> We can probably remove this `console.log`.

D'oh!